cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1907,6 +1907,10 @@ if (roofit_multiprocess)
     add_subdirectory(builtins/zeromq/libzmq)
   endif()
 
+  # zmq_ppoll is still in the draft API, so enable that transitively
+  target_compile_definitions(libzmq INTERFACE ZMQ_BUILD_DRAFT_API)
+  target_compile_definitions(libzmq INTERFACE ZMQ_NO_EXPORT)
```

Comment:
This won't work when `libzmq` is external. Please make sure to test that picking up zeromq and/or cppzmq as external dependencies works as well.